styles.css

```diff
@@ -13,11 +13,11 @@ black - #000000;
 body {
```

Comment:
Remember to delete the commented out code at the top and you could move the .hidden rule declaration to the bottom of the file to avoid using !important. But that is more a matter of preference, so either way works.